README.md

```diff
@@ -1 +1,83 @@
-2020-jun-project1-group2
\ No newline at end of file
+2020-jun-project1-group2
+
+# Project 1
+
+This repository is related to source code of the project 1 of group2.
```

Comment:
We can directly have the description after this. Add the table of contents as well